tests/models/bert/test_modeling_tf_bert.py

```diff
@@ -57,7 +57,7 @@ def __init__(
         use_labels=True,
         vocab_size=99,
         hidden_size=32,
-        num_hidden_layers=5,
+        num_hidden_layers=1,
```

Comment:
Let's not use 1 but 2 if possible. `1` is kind exceptional.

Reply:
Fixed!